gifted_build/src/components/footer.tsx

```diff
@@ -5,8 +5,14 @@ class Footer extends React.Component<{}, {}> {
     render () {
         return (
             <div id="footer">
-                Footer
+                <img src={require('../files/instagram.png')}/>
```

Comment:
Instead of using brands as images, you should be using fontawesome logos, as these are directly taken from there and less files will then be needed 

https://fontawesome.com/icons?d=gallery&s=brands

Reply:
this footer should also be flushed to the bottom of the page.

Reply:
Footer is at the bottom, and the brands are no longer images saved to the files.